MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift

```diff
@@ -39,11 +39,12 @@ extension MovieDetailViewController: DesignProtocol {
         contentView.addSubview(review)
         
         recommendationsView = RecommendationsView()
-        recommendationsView.selectedMovie = { [weak self] movieId in
-            guard let self = self else { return }
-            
-            self.presenter.selectedMovie(withId: movieId)
-        }
+        recommendationsView
+            .selectedMoviePublisher
+            .sink { [weak self] movieId in
+                self?.presenter.selectedMovie(withId: movieId)
+            }
+            .store(in: &recommendationsView.disposables)
```

Comment:
You can move this to `bindViews` or wherever you're doing subscribing.
Also, you don't need `disposables` to be from the `recommendationsView`, the `sink` can use this viewController's disposables.